lil_aretomo/aretomo.py

```diff
@@ -21,7 +21,8 @@ def align_tilt_series(
         output_pixel_size: Optional[float] = 10,
         nominal_rotation_angle: Optional[float] = None,
         correct_tilt_angle_offset: bool = False,
-        gpu_ids: Optional[Sequence[int]] = None
+        gpu_ids: Optional[Sequence[int]] = None,
+        skip_if_completed: Optional[bool] = False
```

Comment:
```suggestion
        skip_if_completed: bool = False
```

Reply:
Updated readme to reflect this too, will need an update on rln side so as not to break AreTomo if setting as not optional

Reply:
the `Optional` type is a synonym for 'this type or None` - it is different from making an argument optional which is achieved as we've done here, a keyword argument with a default value